### PR TITLE
Adding more configs from KIP-631

### PR DESCRIPTION
**Pull request by mumrah**
Adding:
* registration.heartbeat.interval.ms
* registration.lease.timeout.ms
* metadata.log.dir

Also adding server-kip500.properties with some default values

### CODE REVIEW

File: config/server-kip500.properties [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+# The role of this server. Setting this puts us in kip-500 mode
+process.roles=broker,controller
+
+# The id of the broker. This must be set to a unique integer for each broker.
+broker.id=0
+
+# The connect string for the controller quorum
+controller.connect=3000@localhost:9092
+
+# The id of the controller. This can be the same or different from the broker id.
```
Comment (cmccabe) [inline config/server-kip500.properties:29]: It has to be different than the broker id :)

File: config/server-kip500.properties [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (cmccabe) [inline config/server-kip500.properties:1]: I kind of like `kip500-server.properties` better-- what do you think?

File: config/server-kip500.properties (hunk 1 above)
Comment (mumrah) [reply config/server-kip500.properties:29]: Ah, my mistake. I thought they were in separate ID spaces

File: config/server-kip500.properties (hunk 2 above)
Comment (mumrah) [reply config/server-kip500.properties:1]: Yea, that sounds good to me. Will we eventually include multi-node configs? Similar to the server-N.properties files?

Comment (cmccabe) [reply config/server-kip500.properties]: maybe... I guess what I wanted to emphasize here is the fact that it's a combined controller+broker

**Comment by cmccabe** [comment]
LGTM

**Pull request merged**